### DISCUSSION TITLE
500 error when following get started

### DISCUSSION THREAD

**Discussion by DarrylWalkerJacobs** [iTwin/itwinjs-core#3443] [Q&A] [answered]
Hi itwinjs community,

Myself and a colleague have both separately followed the [web application quick start guide](https://developer.bentley.com/tutorials/web-application-quick-start), using the helpful buttons to automatically create and configure the web app registration and iModel.

Once `.env` is configured, localhost boots fine but will not display the model in the window. Additionally, checking the network inspector shows a 500 error on endpoint ending `IModelTileRpcInterface-3.0.0-requestTileTreeProps`.

The endpoint gives the following JSON response:

```
{
   "message":"TreeId=1b_d-E:0_0x2000000000e",
   "errorNumber":65559,
   "name":"Invalid Id",
   "isError":true
}
```

This happens for two different iModels.

This happens in both Edge `99.x` and Safari `15.2`. My colleague sees identical errors.

Are we doing something wrong? Is this normal/expected (e.g. there is some latency while the new iModel propagates)?

Appreciate any help or advice you can give. Screenshots below.

<img width="1335" alt="Screenshot 2022-03-30 at 14 50 35" src="https://user-images.githubusercontent.com/94604001/160850455-cc972075-1f68-4d2d-8e1b-e4f0dac4caff.png">

<img width="3008" alt="Screenshot 2022-03-30 at 14 40 10" src="https://user-images.githubusercontent.com/94604001/160849490-cc31dc8f-f5d7-4998-90f3-e6fcf84aa7b0.png">

<img width="771" alt="Screenshot 2022-03-30 at 14 40 16" src="https://user-images.githubusercontent.com/94604001/160849850-ce1e3b44-85cc-43bc-abd6-247bba983a96.png">

**Comment by aruniverse** [comment, marked as answer]
Hi! Sorry to hear you're having issues right out the bat. Can you confirm what version of iTwin.js you're using? You can click that iTwin.js logo in the bottom left corner and it should pop up some info including the current version number.

**Reply by DarrylWalkerJacobs** [reply]
Hi there!

I am using iTwin.js `v3.1.0`. This was automatically installed by the helpful CRA template that is provided.

Thanks!

**Reply by aruniverse** [reply]
Hey sorry about that, we had some issues with our backend; everything should be resolved now, can you try running your viewer again?

**Reply by DarrylWalkerJacobs** [reply]
Thanks, can confirm the issue is resolved and the models load as expected!
